jujugui/static/gui/src/test/test_app.js

```diff
@@ -1488,6 +1571,7 @@ describe('App', function() {
         jujuCoreVersion: '2.0.0',
         user: userClass
       });
+      app.controllerAPI.connect();
```

Comment:
Should state not be dispatching so this not required (And below)? Do you know why it's required here?